src/main/java/kr/codesqaud/cafe/controller/ArticleController.java

```diff
@@ -92,11 +96,17 @@ public Reply saveReply(@PathVariable int index, @RequestParam String contents,
         return articleService.writeReply(index, contents, sessionDto.getName());
     }
 
-    @DeleteMapping("/article/{articleIndex}/delete/{index}")
+    @DeleteMapping("/article/deleteReply/{index}")
```

Comment:
```suggestion
    @DeleteMapping("/article/reply/{index}")
```

- 겹치는 url 은 없는 것 같습니다.
- 메서드에 delete 를 표현하셨습니다.

어떤 이유로 url 에도 행위를 표현하신건가요?

Reply:
네이밍에 대해서 깊게 생각을 안하고 코드를 쳐서 신경쓰지 못했습니다.
REST API 컨벤션에 대해 더 공부 해보겠습니다.